tests/providers/google/cloud/operators/test_dataprep.py

```diff
@@ -26,14 +26,15 @@
 INCLUDE_DELETED = False
 EMBED = ""
 DATAPREP_JOB_RECIPE_ID = 1234567
+PATH_TO_OUTOUT_FILE = "path_to_output_file"
```

Comment:
If you use same variables in example DAG then it would be wise to import them from the example to keep test and example in consistency.